packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -116,6 +117,28 @@ export class Meter implements types.Meter {
     return new GaugeMetric(name, opt);
   }
 
+  /**
+   * Provide a pre-computed re-useable LabelSet by
+   * converting the unordered labels into a canonicalized
+   * set of lables with a unique labelSetKey, useful for pre-aggregation.
+   * @param labels user provided unordered Labels.
+   */
+  labels(labels: types.Labels): types.LabelSet {
+    let keys = Object.keys(labels).sort();
+    let labelSetKey = keys.reduce((result, key) => {
+      if (result.length > 2) {
+        result += ',';
+      }
+      result = result + key + ':' + labels[key];
+      return result;
+    }, '|#');
```

Comment:
Generate labelSetKey as "|#key1:val1,key2:val2", not sure if we need the prefix "|#", just be consistent with GO [here](https://github.com/open-telemetry/opentelemetry-go/blob/440b6653c9f8069bc6cf26a21549f04fd6859cd9/sdk/metric/sdk.go#L259-L266).